src/lib/Confetti.svelte

```diff
@@ -1,9 +1,9 @@
 <script>
   import { onMount } from "svelte"
 
-  export let size = 10
-  export let x = [-0.5, 0.5]
-  export let y = [0.25, 1]
+  export let size = 10;
```

Comment:
Looking good, thanks for doing this.

Reply:
No problem

Reply:
You sure?

Reply:
Yep

Reply:
Again!

Reply:
Acknowledged

Reply:
The published objects weren't stateful!

Reply:
Are they stateful now?

Reply:
Yeah, fixed!  The only challenge now is that `refresh` doesn't update the thread state for some reason.